assets/src/modules/map.js

```diff
@@ -180,20 +180,30 @@ export default class map extends olMap {
                 let minResolution = node.wmsMinScaleDenominator <= 1 ? undefined : Utils.getResolutionFromScale(node.layerConfig.minScale, metersPerUnit);
                 let maxResolution = node.wmsMaxScaleDenominator <= 1 ? undefined : Utils.getResolutionFromScale(node.layerConfig.maxScale, metersPerUnit);
 
+                // The layer is configured to be cached
                 if (node.layerConfig.cached) {
+                    // Using WMTS
                     const parser = new WMTSCapabilities();
                     const result = parser.read(lizMap.wmtsCapabilities);
+                    // Build WMTS options
                     const options = optionsFromCapabilities(result, {
                         layer: node.wmsName,
                         matrixSet: mainLizmap.projection,
                     });
```

Comment:
```suggestion
                    let options;

                    // Build WMTS options
                    if (result['Contents']['Layer']) {
                        options = optionsFromCapabilities(result, {
                            layer: node.wmsName,
                            matrixSet: mainLizmap.projection,
                        });
                    }
```

We need this because of this fix in progress: https://github.com/openlayers/openlayers/pull/15525